pkg/vm/vm_ops_maths.go

```diff
@@ -94,6 +94,45 @@ func Dec(op stack.Instruction, ctx *stack.Context, istack *stack.Invocation, rst
 	return NONE, nil
 }
 
+// Nz returns a stack Item whose value is:
```

Comment:
Nz pops an integer from the stack.
Then pushes a boolean to the stack which evaluates to true
iff the integer was zero
Returns an error if the popped item cannot be casted to an integer
or if we cannot create a boolean

Reply:
> iff the integer was zero

you mean:

iff the integer was **not** zero

correct?

Reply:
Yep sorry, `evaluates to true iff the integer was non-zero`

Reply:
:+1: